### PR TITLE
Add agent release management team to .ddqa config

### PR DESCRIPTION
**Pull request by kacper-murzyn**
### What does this PR do?

This PR adds agent-release-management team to .ddqa config file so QA cards can be generated properly during Agent releases.

### Motivation

Agent QA consistency.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by amenasria — APPROVED** [review]
LGTM !

**Comment by bkabrda** [comment]
I don't feel like agent-release-management belongs there. Yes, datadog-agent is one of the repositories that it affects, but it now used DD-wide (used for releases of multiple projects) and we review/verify all changes very cautiously before merging them.

**Comment by KSerrania** [comment]
> I don't feel like agent-release-management belongs there. Yes, datadog-agent is one of the repositories that it affects, but it now used DD-wide (used for releases of multiple projects) and we review/verify all changes very cautiously before merging them.

@bkabrda I believe there is some confusion here - this PR is about the Agent Release Management _team_ (which, currently, is equal to Kacper), not the `agent-release-management` GitHub repository.

File: .ddqa/config.toml [hunk 1]
```diff
@@ -262,3 +262,14 @@ jira_statuses = [
...
+[teams."Agent Release Management"]
+jira_project = "AGNTR"
+jira_issue_type = "Task"
+jira_statuses = [
+  "To Do",
+  "In Progress",
+  "Done",
+]
+github_team = "agent-release-management"
+github_labels = ["team/agent-release-management "]
```
[the remark this answers is not in the document]
Comment (kacper-murzyn) [reply .ddqa/config.toml]: It was not, thanks for spotting that.

**Comment by bkabrda** [comment]
Oh, that's a nice oversight that I did right there :D please ignore me :)

**Pull request merged**